ext/intl/calendar/calendar.stub.php

```diff
@@ -488,6 +488,10 @@ public function toDateTime(): DateTime|false {}
 /** @not-serializable */
 class IntlGregorianCalendar extends IntlCalendar
 {
+    public static function createFromDate(int $year, int $month, int $dayOfMonth): static {}
+
+    public static function createFromDateTime(int $year, int $month, int $dayOfMonth, int $hour, int $minute, ?int $second = null): static {}
```

Comment:
Same comment about the other Intl method

Reply:
The same issue :)